deploy/kubernetes/csi-snapshotter/priv-rbac-csi-snapshotter.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: csi-snapshotter-priv
```

Comment:
we dont need a different service accound as they are namespace, we could use the same service account just a new ClusterRole and binding